poetry.lock

```diff
@@ -0,0 +1,3225 @@
+[[package]]
+name = "alabaster"
+version = "0.7.12"
```

Comment:
so this is autogenerated and is considered for the latest version of testing right?

Reply:
Yes, but see [my comment below](https://github.com/ibis-project/ibis/pull/2937#issuecomment-921156067).

Reply:
You can think of the lock file as a sort dumb cache for poetry's dependency solver. It's a record of what dependencies were produced at the time of the solve.